minimalkv/net/s3fsstore.py

```diff
@@ -76,3 +82,132 @@ def _url_for(self, key) -> str:
         return self._fs.url(
             f"{self.bucket.name}/{self.object_prefix}{key}", expires=self.url_valid_time
         )
+
+    def __eq__(self, other):
+        """
+        Assert that two ``S3FSStore``s are equal.
```

Comment:
I wouldn't call it assert, as there is no assert statement in the function body.